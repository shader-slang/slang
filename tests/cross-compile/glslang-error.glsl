//TEST(glslang):SIMPLE:-pass-through glslang -target spirv -entry compute -stage compute -profile sm_5_1 
#version 450

void main()
{
    int a = !;
}
