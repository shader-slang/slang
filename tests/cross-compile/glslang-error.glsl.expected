result code = -1
standard error = {
glslang: ERROR: tests/cross-compile/glslang-error.glsl:6: '' :  syntax error, unexpected SEMICOLON
ERROR: 1 compilation errors.  No code generated.

}
standard output = {
}
